### DISCUSSION TITLE
Mockito Inline?

### DISCUSSION THREAD

**Discussion by rdehuyss** [mockito/mockito#3275] [Q&A]
With mockito-inline not being updated anymore, what's the way forward to mock static methods on Java 8?

I tried mockito-subclass but that didn't work...

**Comment by TimvdLippe** [comment]
If you need support for Java 8, please use Mockito 4. Mockito 5 requires Java 11+.

**Reply by rdehuyss** [reply]
Hmm, perhaps I should reformulate: the library, [JobRunr](https://github.com/jobrunr/jobrunr) must support Java 8 but I use Java 11 in the test sources using Gradle.

**Reply by TimvdLippe** [reply]
If you must run on Java 8, then you need to use Mockito 4. You can use Mockito 4 in Java 11 as well.
